recipes/bioconductor-dexseq/build.sh

```diff
@@ -5,6 +5,11 @@ $R CMD INSTALL --build .
 
 python_scripts=$PREFIX/lib/R/library/DEXSeq/python_scripts
 
+if [ "$PY3K" == 1 ]; then
+    2to3 -w $python_scripts/dexseq_count.py
+    2to3 -w $python_scripts/dexseq_prepare_annotation.py
+fi
```

Comment:
It seem that this collide with the sed command a few lines below on OSX.
Maybe patching both scripts would be more clean?

Reply:
Ok, do you mean I should submit py3 changes for these scripts upstream to the dexseq authors? (I don't know that much python but I guess I could try)

Reply:
Ideally yes, but the patch can also be included here for the time being.